.htmllintrc

```diff
@@ -2,6 +2,6 @@
     // names of npm modules to load into htmllint
     "plugins": [],
 
-    "attr-bans": "none",
+    "attr-bans": false,
```

Comment:
@two24studios - just a heads up `none` was throwing errors in an html linter, but `false` passes.